h/buildext.py

```diff
@@ -303,6 +304,13 @@ def check_sentry_dsn_is_public(dsn):
 parser.add_argument('browser',
                     help='Specifies the browser to build an extension for',
                     choices=['chrome'])
+parser.add_argument('--bouncer-url',
```

Comment:
The existing parameters which take urls are `--service` and `--websocket`, without a `-url` suffix. So this should be called `--bouncer`, and mapped to `args.bouncer_url` using the `dest` keyword argument.

It should also be defined with the `--options`, not down here with the positional arguments.